packages/communication-react/review/beta/communication-react.api.md

```diff
@@ -2096,6 +2099,7 @@ export const DEFAULT_COMPOSITE_ICONS: {
     CaptionsIcon: JSX.Element;
     CaptionsOffIcon: JSX.Element;
     CaptionsSettingIcon: JSX.Element;
```

Comment:
BTW, let's name it consistently here - if we call it CaptionsSetting then we call it all CaptionsSetting, if we call it CaptionsSetting*s* then we add all s to the end

I've seen Settings a lot from other UI, maybe we could just name it all as CaptionsSetting*s*